sections/sponsor-addons/sponsor-addons.tsx

```diff
@@ -1,24 +1,22 @@
-import styles from './addons.module.scss';
+import styles from './sponsor-addons.module.scss';
 import Card from '@components/card/card';
 import ButtonLink from '@components/button-link/button-link';
 import { H3, Text } from '@components/typography/typography';
 
 const Addons = () => (
   <section className={styles['sponsor-addons']}>
     <div className={styles['sponsor-addons-content']}>
-      <H3>Addons</H3>
+      <H3 className={styles['addons-header']}>Addons</H3>
+      <Text>Don’t want to pick favourites? You don’t have to!<br />
+       Bring additional mentors and guests to assist students and experience all that UNIHACK has to offer.</Text>
       <div className={styles.addons}>
         <Card className={styles['addon-card']}>
           <Text>Additional Mentors</Text>
-          <Text bold>$250</Text>
-        </Card>
-        <Card className={styles['addon-card']}>
-          <Text>Additional Guests</Text>
-          <Text bold>$150</Text>
+          <span className={styles['right-text']}> <Text bold>$250</Text></span>
```

Comment:
this can just be `<Text bold>...</Text>` and we can use `justify-content: space between` on the div to handle the layout - no need for floats